app/components/search-facet-language/component.js

```diff
@@ -43,7 +42,8 @@ export default Ember.Component.extend({
     actions: {
         changeFilter(languageNames) {
             let key = this.get('key');
-            let [filter, value] = this.buildQueryObject(languageNames);
+            let selected = Ember.$.isArray(languageNames) ? languageNames : [languageNames];
```

Comment:
Since I switched from `power-select-multiple` to `power-select`, it'll just be a string when the user selects a language.